spec/controllers/rebate_forms_controller_spec.rb

```diff
@@ -32,7 +32,33 @@
     end
   end
 
-  pending '#update'
+  describe '#update' do
```

Comment:
This is the controller for the api. We shouldn't be able to update from the api.